libconsensus/pbft/PBFTReqCache.cpp

```diff
@@ -80,6 +80,31 @@ bool PBFTReqCache::generateAndSetSigList(dev::eth::Block& block, IDXTYPE const&
     return false;
 }
 
+void PBFTReqCache::addViewChangeReq(ViewChangeReq const& req, int64_t const&)

```

Comment:
if the second parameter is useless, why not delete it ?

Reply:
This will be used in the future to solve consensus-problem